README.md

```diff
@@ -33,89 +33,92 @@ System.Web.Bem - БЭМ-инфрастурктура для ASP.NET MVC.
-- подключается C# библиотека `System.Web.Bem` для серверной шаблонизации во время работы приложения.
+**System.Web.Bem** is the BEM-infrastructure for ASP.NET MVC projects. During the installation of [System.Web.Bem package](https://www.nuget.org/packages/System.Web.Bem) into the project:
+- folder `BEM` with the files structure is created;
+- assembler of BEM-projects [enb](https://ru.bem.info/toolbox/enb) is installed from npm and proper configuration file for enb is created;
+- reference for .NET library named System.Web.Bem is added (this library allows to use BEMHTML templates on the server side).
 
-### Структура проекта
+### Project structure
 
 ```
-<Project root>                  // корневая папка проекта
-├─ Bem                          // папка БЭМ-проекта
+<Project root>
+├─ Bem                          // BEM-files folder
 │  ├─ .enb
-│  │  └─ make.js                // конфиг сборщика enb
-│  ├─ desktop.blocks            // уровень переопределения, внутри находятся блоки
+│  │  └─ make.js                // enb configuration file
+│  ├─ desktop.blocks            // redefinition level (blocks are located inside)
 │  │  ├─ block-1 
 │  │  ├─ block-2 
 │  │  │  ... 
 │  │  └─ block-n 
-│  │     ├─ block-n.bemhtml.js  // реализация блока block-n в технологии bemhtml.js (шаблоны)
-│  │     ├─ block-n.css         // реализация блока block-n в технологии css 
+│  │     ├─ block-n.bemhtml.js  // implementation of 'block-n' in 'bemhtml.js' technology (templates)
+│  │     ├─ block-n.css         // implementation of 'block-n' in 'css' technology
 │  │     │  ...                 // ...
 │  │     └─ block-n.js
-│  ├─ desktop.bundles           // папка с бандлами проекта
+│  ├─ desktop.bundles           // folder for the bundles
 │  │  ├─ bundle-1 
 │  │  ├─ bundle-2 
 │  │  │  ... 
 │  │  └─ bundle-n 
-│  │     └─ bundle-n.bemdecl.js // декларация бандла bundle-n 
-│  └─ levels.js                 // список уровней переопределения
+│  │     └─ bundle-n.bemdecl.js // declaration of the bundle named 'bundle-n'
+│  └─ levels.js                 // list of redefinition levels
 │  ...
-├─ Controllers                  // Controllers, Models, Views - стандартные папки ASP.NET MVC
+├─ Controllers                  // Controllers, Models, Views - usual ASP.NET MVC folders
 ├─ Models
 ├─ Views
 │  ...
-├─ package.json                 // конфиг npm
-└─ Web.config                   // конфиг вашего приложения
+├─ package.json                 // npm configuration file
+└─ Web.config                   // configuration file of your application
 ```
 
-### Сборка
-**Внимание! Для сборки проекта необходимо, чтобы на компьютере был установлен [node.js](https://nodejs.org/en/). Для работы приложения на сервере устанавливать node.js нет необохдимости.**
+### Build the bundles
+**Note! Node.js is required for project building. Node.js on the production server is not required.**
```

Comment:
-> [node.js](https://nodejs.org/en/) must be installed for building the project. Node.js is not required on the production server.